.circleci/config.yml

```diff
@@ -276,6 +267,29 @@ jobs:
     executor: python27
     steps:
       - test_build
+  test_build_win_py38: &test_build_win
+    executor:
+      name: win/default
+      size: *resource_class
+      shell: bash.exe
+    environment:
+      PYTHON_VERSION: 3.8.3
```

Comment:
does it have to be exact? are we able to say "latest"?

Reply:
I chose to use a specific version as that's what we do in the Dockerfile. We can't be entirely consistent across Linux and Windows builds though, as currently implemented, since nuget doesn't have Windows packages published for more recent releases of 3.6 and 3.5.